src/main/java/frc/robot/utils/dynamicauto/SwapHeadings.py

```diff
@@ -0,0 +1,26 @@
+import json
+
+# open file in write mode
+with open('src/main/java/frc/robot/utils/dynamicauto/data.chor', 'r+') as file:
+    data = json.load(file)
+
+    """
+    Things to Change for each path
+    1. Clear Trajectory array
+    2. Reverse headings for each waypoint
```

Comment:
why do we need this

Reply:
sorry was my internal thoughts.. we can remove